docs/DEPLOYMENT.md

```diff
@@ -255,7 +281,6 @@ metadata:
   labels:
     app: tekton-hub-api
 data:
-  CONFIG_FILE_URL: https://raw.githubusercontent.com/tektoncd/hub/master/config.yaml
   CATALOG_REFRESH_INTERVAL: 30m ## change the catalog refresh interval here
```

Comment:
should we do something like
```yaml
data:
     CATALOG_REFRESH_INTERVAL: 0m
     ...
```

Reply:
> should we do something like
> 
> ```yaml
> data:
>      CATALOG_REFRESH_INTERVAL: 0m
>      ...
> ```

cc @piyush-garg @PuneetPunamiya

Reply:
I think we dhould keep defualt time interval to 30
Any reason you were suggesting to 0m ?

Reply:
@PuneetPunamiya @pratap0007 my bad, was just suggesting that since other fields which are actually part of configmap are missed so should we just add `...` in place of that?
```
data:
    CATALOG_REFRESH_INTERVAL: 30m
    ...
```